code/game/gamemodes/cult/cult_items.dm

```diff
@@ -74,10 +74,10 @@
 	knockdown_duration = 2 SECONDS
 
 /obj/item/restraints/legcuffs/bola/cult/throw_at(atom/target, range, speed, mob/thrower, spin, diagonals_first, datum/callback/callback)
-	. = ..()
 	if(!iscultist(thrower))
 		thrower.visible_message("<span class='danger'>The bola glows, and boomarangs back at [thrower]!</span>")
 		throw_impact(thrower)
+	. = ..()
```

Comment:
will this still cause the bola to be thrown?

Reply:
Re-tested just to be sure

Yeah - If you're a cultist, its the same functionality as always, normal bola behavior
If you're -not- a cultist, doesn't throw any bola, thus preventing another from being bola'd if next to you tile-wise, it just instantly snaps it into your leg for throwing cultbola as non cult, so should be good

Reply:
sick